### ISSUE TITLE
Can't craft any items in Refined Storage crafting grid 

### ISSUE THREAD

**Issue by Jocky69** [FTBTeam/FTB-Modpack-Issues#2120]
### Modpack

FTB Stoneblock 3

### Modpack version

1.4.0

### Describe the bug

Trying to craft an item in the Refined Storage crafting grid, there's an '' ! '' instead of the '' + '' sign beside each recipe saying '' Move item is not supporting with this recipe and this container ''

### Steps to reproduce

1.Open crafting grid
2.Click on a recipe
2.The message appear with a '' ! '' instead of a '' + '' sign

### Expected behaviour

Supposed to be a '' + '' sign instead of a '' ! ''

**Comment by manmaed** [comment]
Can you please test if this fixed in 1.4.1

**Comment by Jocky69** [comment]
Yes, thanks for this quick response and fixing the bug in the server too. 😉. Thanks again.
